astropy/visualization/tests/test_interval.py

```diff
@@ -1,14 +1,12 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-import pytest
 import numpy as np
+import pytest
 
 from astropy.utils import NumpyRNGContext
-from astropy.visualization.interval import (ManualInterval,
-                                            MinMaxInterval,
-                                            PercentileInterval,
-                                            AsymmetricPercentileInterval,
-                                            ZScaleInterval)
+from astropy.visualization.interval import (
+    AsymmetricPercentileInterval, ManualInterval, MinMaxInterval, PercentileInterval,
+    ZScaleInterval)
```

Comment:
bleugh

Reply:
Isn't that going to change with Black?

Reply:
oh yeh maybe. Why aren't we already using the black isort profile?